src/components/molecules/Visualizer/Engine/Cesium/Cluster.tsx

```diff
@@ -25,51 +25,63 @@ const Cluster: React.FC<ClusterProps> = ({ property, children }) => {
     clusterImageHeight,
   } = property?.default ?? {};
 
-  const cluster = useMemo(() => {
-    return new EntityCluster({
+  const cluster = useRef(
+    new EntityCluster({
       enabled: true,
       pixelRange: clusterPixelRange,
-      minimumClusterSize: clusterMinSize,
+      minimumClusterSize: 2,
       clusterBillboards: true,
       clusterLabels: true,
       clusterPoints: true,
-    });
-  }, [clusterMinSize, clusterPixelRange]);
+    }),
+  );
```

Comment:
it looks like the useRef is being used incorrectly; there is no need to use it.

The performance is also worse because new EntityCluster is executed every time even though useRef is used. If this is the case, implement it without useRef first, and then consider how you can optimize the performance.

```ts
const cluster = useRef(
    new EntityCluster({
      // ...
    })
);
```

=>

```ts
  const culuster = new EntityCluster({
      enabled: true,
      pixelRange: clusterPixelRange,
      minimumClusterSize: clusterMinSize,
      minimumClusterSize: 2,
      clusterBillboards: true,
      clusterLabels: true,
      clusterPoints: true,
    });
```

Reply:
@rot1024 , 
Can I use `useMemo` with empty array (so it should compute once?) like this:

```
const cluster = useMemo(() => {
    return new EntityCluster({
      enabled: true,
      pixelRange: 15,
      minimumClusterSize: 2,
      clusterBillboards: true,
      clusterLabels: true,
      clusterPoints: true,
    });
  }, []);
```
or any suggestions.

Reply:
OK give it a shot.